tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -276,3 +308,13 @@ def _a11y_click_menu_item(self, menu_item_locator):
     def _wait_for_toggle_ready(self, by, locator):
         checkbox = self.marionette.find_element(by, locator)
         Wait(self.marionette).until(expected.element_enabled(checkbox))
+
+    # this method is a copy of go_back() method in regions/keyboard.py
+    def return_to_main_menu(self, gaia_header=True):
+        # TODO: remove tap with coordinates after Bug 1061698 is fixed
+        if gaia_header:
+            self.marionette.find_element(*self._header_locator).tap(25, 25)
+        else:
+            self.marionette.find_element(*self._non_gaia_header_locator).tap(25, 25)
```

Comment:
Ah, but if I do it that way when they have non-gaia header, I won't find it out.  ( I think they have to use gaia header for the settings menu)